tests/all-tests.js

```diff
@@ -41,7 +41,6 @@ var testIncludes = [
     "./PayloadSizeTest.js",
     "./PSPIntegrationTests.js",
     "./ResetAtStartTests.js",
-    "./StartupAPITests.js",
```

Comment:
To avoid wrongfooting people, please delete the entire file as well as not referencing it here

Reply:
Done.